stats/Export.md

```diff
@@ -0,0 +1,34 @@
+# Export API Overview
+The stats library aggregates measurements into Views (which are essentially a collection of
+metrics, each with a different set of labels). All output objects are immutable and read-only.
+The core data types used are:
+* `AggregationData`: contains data from an `Aggregation`.
+* `ViewData`: encapsulates `View` data aggregation.
+
+### AggregationData
+An AggregationData describes the result of data aggregation. The library SHOULD define an
+AggregationData for each Aggregation types defined.
+
+The library SHOULD provide support for multiple types of Aggregations:
+* `CountData`: data generated for a `Count` aggregation.
+* `SumDataDouble` and `SumDataInt`: data generated for a `Sum` aggregation based on the `Measure`
```

Comment:
s/SumDataInt/SumDataInt64 or SumDataLong?

Reply:
Probably Int64; Long is more language-specific (and in C++ is often only 32 bits).

Reply:
done.